src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -64,7 +70,14 @@ extends AbstractMap[Key, Value]
   private[this] val actualInitialSize = OpenHashMap.nextPositivePowerOfTwo(initialSize)
 
   private var mask = actualInitialSize - 1
-  private var table : Array[Entry] = new Array[Entry](actualInitialSize)
+
+  /** The hash table.
+    * 
+    * The table's entries are initialized to `null`, indication of an empty slot.
+    * A slot is either deleted or occupied if and only if the entry is non-`null`.
```

Comment:
Maybe better: "A slot is either deleted or occupied. It is occupied if and only if the entry is non-`null` and entry's keys is non-`null`."